### PR TITLE
docs: edits to JX 3 Getting Started

### PR DESCRIPTION
**Pull request by MarckK**

### CODE REVIEW

File: content/en/docs/v3/getting-started/_index.md [hunk 1]
```diff
@@ -8,28 +8,28 @@ lastmod: 2020-02-21
...
+- __Secrets Management__ - Jenkins X requires a number of secrets, some are generated and some are provided at installation by the user. These need to be managed and there are a number of solutions that help.  Jenkins X prefers to use managed cloud services where possible: [Google Secrets manager](https://cloud.google.com/secret-manager), [AWS Secrets manager](https://aws.amazon.com/secrets-manager/) are good examples where secrets are stored out of the cluster and syncronsied in cluster using [external secrets](https://github.com/godaddy/kubernetes-external-secrets).  Where managed cloud services are not available or desired, Jenkins X can also use Vault.
```
Comment (ankitm123) [inline content/en/docs/v3/getting-started/_index.md:33]: So AWS Secrets Manager is not integrated yet (will be added soon), so may be say something like soon, or in the works?

File: content/en/docs/v3/getting-started/_index.md [hunk 2]
```diff
@@ -8,28 +8,28 @@ lastmod: 2020-02-21
...
 jx upgrade plugins
 ```
 
 ## Pick your initial git repository
 
-Before you chose which Git template you start with here's a quick explanatiom of both:
+Before you chose which Git template you start with, note that part of removing complexity out of Jenkins X 3.x alpha has meant relying on other solutions where appropriate. Here's a quick explanation of our recommendations for setting up Cloud Infrastructure and for Secrets Management with Jenkins X 3:
 
--  __Cloud Infrastructure__ - Jenkins X requires a Kubernetes cluster, storage, networking, DNS, IAM bindings, all things that we refer to cloud resources.  Jenkins X 3.x alpha is recommending to use Terraform + optinally Terrafrom Cloud to create and manage your cloud resources, we have quickstarts and guides to help.
+-  __Cloud Infrastructure__ - Jenkins X requires a Kubernetes cluster, storage, networking, DNS, IAM bindings, all things that we refer to as cloud resources. We recommend you use Terraform, or optinally Terrafrom Cloud, to create and manage your cloud resources for Jenkins X 3.x alpha. To help you set up your Cloud Infrastructure, we have created quickstarts and guides.
```
Comment (ankitm123) [inline content/en/docs/v3/getting-started/_index.md:31]: typo for `optinally`

File: content/en/docs/v3/getting-started/_index.md (hunk 1 above)
Comment (MarckK) [reply content/en/docs/v3/getting-started/_index.md:33]: I'll take out the reference to AWS, as this is general explanatory text and not the main AWS instructions. The reference to AWS will need to be updated soon, and might drift apart from the main AWS instructions.

**Pull request merged**